.github/workflows/docker-nightly-build.yml

```diff
@@ -0,0 +1,41 @@
+name: Docker Image Nightly Build CI
+
+on:
+  push:
+    branches: [ cron_build ]
+  
+  schedule:
+    # run daily 0:00 
+    - cron: '0 0 * * *'
+
+env:
+  ORG: timescaledev
+
+jobs:
+
+  # Build ubuntu TimescaleDB images for both TSL and OSS code.
+  timescaledb:
+
+    name: PG${{ matrix.pg }}${{ matrix.oss }}
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        pg: [12, 13 ]
+
+    steps:
+    - uses: actions/checkout@v2
+      
+    - name: Set up Docker Buildx
+      id: buildx
+      uses: docker/setup-buildx-action@v1
+
+    - name: Linux available buildx platforms 
+      run: echo ${{ steps.buildx.outputs.platforms }}
+
+    - name: Login to DockerHub Registry
+      run: echo ${{ secrets.DOCKERHUB_TOKEN }} | docker login -u ${{ secrets.DOCKERHUB_USER }} --password-stdin
+
+    - name: Build and push nightly Docker image for TimescaleDB
+      run: make nightly ORG=$ORG PG_VER=pg${{ matrix.pg }}
```

Comment:
Don't you need to handle both OSS and TSL version when you build them?

Reply:
removed oss.